README.md

```diff
@@ -30,9 +30,12 @@ We note all the input datasets are publicly available using a NASA Earthdata acc
     machine urs.earthdata.nasa.gov
         login <username>
         password <password>
-    ```
-    The `username`/`password` are the appropriate Earthdata Login credentials that are used to access NASA data. This file is necessary for downloading the Sentinel-1 orbit files from the ASF DAAC. Additionally, the [`requests`](https://docs.python-requests.org/en/latest/) library automatically uses credentials stored in the `~/.netrc` for authentification when none are supplied.
 
+    machine dataspace.copernicus.eu
+        login <username>
+        password <password>
+    ```
+    The first `username`/`password` pair are the appropriate Earthdata Login credentials that are used to access NASA data. The second pair are your credentials for the [Copernicus Data Space Ecosystem](https://dataspace.copernicus.eu). This file is necessary for downloading the Sentinel-1 files, and auxiliary data. Additionally, the [`requests`](https://docs.python-requests.org/en/latest/) library automatically uses credentials stored in the `~/.netrc` for authentification when none are supplied.
```

Comment:
Thank you for providing this information! Will add this shortly. Do we need to add additional secrets to this github org or repo? Has it been done?

Reply:
@cmarshak all the tests passed, so there doesn't appear to be a need to add these secrets to the repo/org. 

On the HyP3 side, we'll add `ESA_USERNAME` and `ESA_PASSWORD` to the secrets manager and they will be available in the container as environment variables.